apps/em/emhrs-cron/aat.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: emhrs-cron
+spec:
+  releaseName: emhrs-cron
+  values:
+    job:
+      disableActiveClusterCheck: true
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
```

Comment:
May be worth moving these properties to the `emhrs-cron.yaml` file, you then won't have to specify them in all of your `<env>.yaml` files.